### ISSUE TITLE
Notice: EMF 1.0 and old issues

### ISSUE THREAD

**Issue by Traben-0** [Traben-0/Entity_Model_Features#113]
EMF 1.0 is practically a different mod to the last beta 0.2.13 and the old private alpha,
Almost the entire model creation code is different and almost all issues reported using old beta versions will be irrelevant going forwards. (especially if it is about broken models)

With that being said, many GitHub issues have been left mostly ignored by me during developement if I felt the issue was something that I expected would get resolved passively during development.

With 1.0's release I will be going through these older issues in the coming months to actually address and weed out which issues are still relevant or not.

# tl;dr What to do
If you have an issue with version 1.0 or later **PLEASE** make a new issue rather than comment in an issue thread started during the beta.
I encourage you to #mention the old issue thread if it is relevant to your new 1.0 issue instead.

**all issues after this post should be for the release version only**
